kotlin-mui-base/src/jsMain/generated/mui/base/Button.classes.kt

```diff
@@ -17,3 +19,10 @@ external interface ButtonClasses {
     /** State class applied to the root `button` element if `focusVisible={true}`. */
     var focusVisible: ClassName
 }
+
+external object buttonClasses : ButtonClasses {
```

Comment:
`external val buttonClasses : ButtonClasses` will work without redundant overrides

Reply:
Also - separate file for constants is preferrable

Reply:
> `external val buttonClasses : ButtonClasses` will work without redundant overrides

Thank you, fixed.

> Also - separate file for constants is preferrable

I believe it's better to place the constant in the same file, the same manner as we do for components (they have their `Props` interfaces in the same file), so hope you're fine with it